components/timeline/vis-timeline.service.ts

```diff
@@ -385,7 +383,7 @@ export class VisTimelineService {
    *
    * @memberOf VisTimelineService
    */
-  public getWindow(visTimeline: string): { start: Date, end: Date } {
+  public getWindow(visTimeline: string): { start: Date; end: Date } {
```

Comment:
In the other projects we reformatted through separate PRs authored by vis-bot. It doesn't give attribution to people for running `prettier --write`, follows one PR one thing filosophy and would also make reviewing easier by filtering out code style changes.

Reply:
I also would seperate style changes from other thinks. But I'm not sure if that's the case here. Also this is such a minor change that I have no problem with it.